atomic_reactor/plugin.py

```diff
@@ -378,9 +378,6 @@ def _translate_special_values(self, obj_to_translate):
             'BUILD_SOURCE_PATH': self.workflow.builder.source.path,
         }
 
-        if self.workflow.builder.base_image:
-            translation_dict['BASE_IMAGE'] = self.workflow.builder.base_image.to_str()
```

Comment:
What was this for, and why do we not need it anymore?

Reply:
in osbs/inputs in those orchestrator/worker inner files, this MACRO could be used, but we don't used that anymore, as we can access base_image from workflow in every plugin